andela/travel_tool_front
# 159271560 Setup docker for the frontend application

#### What does this PR do?
It provides a Dockerfile that enables the application to run inside a container.

#### Description of Task to be completed?
Create a Dockerfile that will enable the application to run inside docker container.

#### How should this be manually tested?
Open up the terminal and navigate into the project directory. Run the following commands:
- `docker build -t travel_tool_front .`
- `docker run -it -p 5000:5000 --rm travel_tool_front`
Open up the browser and head to `localhost:5000`, you shall see the application running.

#### Any background context you want to provide?
Before running the docker commands, you would need to install node packages and build the application:
`npm install`
`npm run build`

#### What are the relevant pivotal tracker stories?
#159271560

#### Screenshots (if appropriate)
N/A

#### Questions:
